### ISSUE TITLE
Asset pane accessibility/ergonomics request

### ISSUE THREAD

**Issue by awulkan** [defold/defold#11598]
**Is your feature request related to a problem? Please describe (REQUIRED):**
I would absolutely love if it was possible to toggle single-click to open files in the asset pane. Right now it requires me to double click every item to open them, which gets tedious. I also suffer from RSI, so every unnecessary click that I have to do limits how much I can use the engine overall.

**Describe the solution you'd like (REQUIRED):**
An option to enable the ability to click on an asset once to open it in the editor. Just like other game engines like Godot, and editors like Visual Studio Code works.

I am not asking for this to be the default. I understand that a lot of people are used to, or prefer to double click. But I would be very happy if you could consider adding a simple setting for this.

**Describe alternatives you've considered (REQUIRED):**
I've tried working as much as possible outside of the editor, by using VS Code. But I still have to regularly use the asset pane in the engine for other things, so it's not a good solution.

**Additional context (OPTIONAL):**
An example of what I mean. Clicking once on an asset doesn't do anything.

https://github.com/user-attachments/assets/a7963beb-e66d-4772-a6e3-2eb3a194fc33

However, if you take a look at other game engines, editors, and IDEs, then a single click is all you need.

Godot:

https://github.com/user-attachments/assets/50980f85-836f-4507-89d4-4e5de5a9e4a4 

VS Code:

https://github.com/user-attachments/assets/fb67898e-8c15-4169-a4b4-8479f0cebc5a

Thank you for reading through this! :)

**Comment by JosephFerano** [comment]
@awulkan We could add this option, although I'm thinking of other ways to facilitate this. The thing about the examples that you're demonstrating from Godot and VSCode is that they are different operations; Godot is previewing a node's properties and VSCode is previewing the file which then requires further interaction to open. We don't have a "preview" feature for files from the Asset Browser. I think instead of having an option for it to be the default, perhaps holding down Alt + click automatically opens it.

However, even if we add that, I feel the editor already has workflows that can facilitate what you're trying to accomplish. You can hold down Ctrl + Click to add items or use Shift + Click to select a range. Then you can either right-click -> "Open" or use a keyboard shortcut (ENTER or Ctrl/Cmd+o work) to open them all in one go, rather than having to double-click each item.

Can you try this multiple-selection workflow out and report how it feels? If you still find it a bit too cumbersome, then maybe we can consider the Alt + click idea.

**Comment by awulkan** [comment]
@JosephFerano 

Hey! Thanks for taking your time to respond.
I guess you're right that the actual problem is that the Defold engine doesn't have a "preview state" that you enter with 1 click, like in a lot of other IDEs.
My assumption would be that the only difference with a preview state would be that the tab isn't added to the tab bar until you double click or edit the file. Having this feature would be amazing in my opinion.

Unfortunately it won't help me if I have to use both my keyboard and mouse to open a file. It would just replace the click of one button with the click of another.
Having to reach for both the keyboard and the mouse every time I want to open a file is not very ergonomic for me, especially since I rarely use the keyboard in the first place. I've tried it after it was suggested before.
